analytical/templatetags/piwik.py

```diff
@@ -96,6 +108,11 @@ def render(self, context):
         if getattr(settings, 'PIWIK_DISABLE_COOKIES', False):
             commands.append(DISABLE_COOKIES_CODE)
 
+        if getattr(settings, 'PIWIK_ASK_FOR_CONSENT', False):
+            commands.append(ASK_FOR_CONSENT_CODE)
+            commands.append(FORGET_CONSENT_CODE)
+            commands.append(REMEMBER_CONSENT_CODE)
```

Comment:
I would be okay with folding the contents of these variables into one. Their content is only used together.

Reply:
Changed in c3065fc0c96e762f064176f16554af440055cadc